packages/apktool.vm/tools/chocolateyinstall.ps1

```diff
@@ -4,17 +4,24 @@ Import-Module vm.common -Force -DisableNameChecking
 try {
   $toolName = 'apktool'
   $category = 'Java & Android'
-  $shimPath = 'bin\apktool.exe'
+  $rawToolPath = Join-Path ${Env:RAW_TOOLS_DIR} "$toolName"
```

Comment:
We should add `-Resolve` to all `Join-Path` to fail the package early if something goes wrong to help with debugging (sorry I had missed this before)